runtimebp/internal/maxprocs/maxprocs.go

```diff
@@ -74,7 +74,7 @@ var (
 		ubermaxprocs.Set(
 			ubermaxprocs.Min(2),
 			ubermaxprocs.Logger(func(s string, i ...interface{}) {
-				fmt.Fprintf(os.Stderr, s, i...) // contains context
+				fmt.Fprintf(os.Stderr, s+"\n", i...) // contains context
```

Comment:
🔕 string concatenation is actually quite inefficient especially when `s` is long (because it needs to make a bigger string and copy `s` over), so just do `fmt.Fprintln(os.Stderr, "")` after this line is probably better.

Reply:
good point!

Reply:
To my comment above I guess this does open up a possibility the output will get interleaved, but I don't think we'll be running any other goroutines at the time this code runs.